##Analysis Proposal for ee282 Final Project

Author: Karina Brocco French

#Analyzing microbiomes of larval sea stars (Pisaster ochraceus).

I propose to study ontogenetic changes in *Pisaster ochraceus* larval microbiomes. Characterizing the microbiome of early life-stage *P. ochraceus* will allow me to conduct future experiments using transcriptomics to examine what genes involved in maintaining host health are expressed under the varying conditions, and how potential shifts in microbiome and host genetics under toxin exposure may contribute to marine disease. 

I expect the microbiome of *P. ochraceus* will shift in species composition across developmental stages since microbiomes vary with morphology and diet within echinoderm host species. Using the 16s rDNA sequencing data, I will answer the following questions: 

a) What microbial species are present within the host at various stages of larval development? 
b) Are there changes in the relative abundances of the microbial species present in the host at various stages of larval development? 

The data set I will be using to conduct my analysis are the 16s rDNA sequence FastQ files for my own P. ochraceus larval samples prepared by the GHTF. I have acquired these data from the GHTF and am in the process of downloading them for use.

I will analyze the similarity/variation of the composition of the associated larval microbiomes at different stages of host development using an Analysis of Similarity and a Principal Component Analysis. I will  use RStudio software to make ordination plots to examine the beta diversity across my samples. The specific type of ordination plot I will use is the multi-dimensional scaling (MDS) plot to compare the level of similarities in microbial species across my samples. To test the statistical significance of the groupings I may see in the samples, I will carry out a Permutational multivariate analysis of variance (PERMANOVA). 

I already have these data and am now in the process of preparing them for use. Others in my lab have used these techniques on similar data sets in the past, and I will use the analyses and visualizations for a poster presentation for SICB and for my committee meeting at the end of this quarter, so I am facing strong incentives in addition to the class assignment to complete these analyses and data visualizations. 


of variance (PERMANOVA). 
I already have these data and am now in the process of preparing them for use. Others in my lab have used these techniques on similar data sets in the past, and I will use the analyses and visualizations for a poster presentation for SICB and for my committee meeting at the end of this quarter, so I am facing strong incentives in addition to the class assignment to complete these analyses and data visualizations. 
I propose to study ontogenetic changes in *Pisaster ochraceus* larval microbiomes. Characterizing the microbiome of early life-stage *P. ochraceus* will allow me to conduct future experiments using transcriptomics to examine what genes involved in maintaining host health are expressed under the varying conditions, and how potential shifts in microbiome and host genetics under toxin exposure may contribute to marine disea

